sdk/data_cosmos/src/resources/permission/permission_token.rs

```diff
@@ -1,5 +1,6 @@
 use super::AuthorizationToken;
 use azure_core::{auth::Secret, base64};
+use tracing::trace;
```

Comment:
Is this even used that you could just add a `use` and not remove one to, say, `log::trace`?